docs/user/coordinates.md

```diff
@@ -0,0 +1,43 @@
+<!--Copyright (C) SuperDARN Canada, University of Saskatchewan 
+Author(s): Marina Schmidt 
+Modifications:
+
+Disclaimer:
+pyDARN is under the LGPL v3 license found in the root directory LICENSE.md 
+Everyone is permitted to copy and distribute verbatim copies of this license 
+document, but changing it is not allowed.
+
+This version of the GNU Lesser General Public License incorporates the terms
+and conditions of version 3 of the GNU General Public License, supplemented by
+the additional permissions listed below.
+-->
+
+# Coordinate Systems 
+---
+
+pyDARN offers several different coordinate systems for plotting: 
+
+## Range-Time Plots 
+
+**Range Gates**: `Coords.RANGE_GATE` a rectangle section determined by beam width and set distance for each range (nominally 45 km). RAWACF and FITACF data give their parameter values with respect to range gates  
+
+**Slant Range**: `Coords.SLANT_RANGE` is a conversion from range gates to km units.  
+
+**Ground Scatter-Mappped Range**: `Coords.GROUND_SCATTER_MAPPED_RANGE` uses echos from groundscatter to adjust slant-range coordinates to be more accurate (km) based on [Dr. Bill Bristow's paper](https://doi.org/10.1029/93JA01470). Implemented by Dr. Nathaniel Frissell and Francis Tholley from University of Scranton. 
+
+**Geographic** (Coming Soon)
+
+**AACGM** (Coming Soon)
```

Comment:
I dont think either of these apply to RTP plots. 
```suggestion
```

Reply:
Apparently, you can get them DaVitpy and RST provide them so I thought we could as well

Reply:
Aah, right so you plot the range gates geographic or geomagnetic longitude/latitude up the y-axis along the beam you're interested in? I get the idea, but it's kinda only giving you half of the information about the position of the echoes, but I guess it's useful to someone.

Reply:
I've amended the suggestion accordingly :P